src/behavior.js

```diff
@@ -59,6 +59,10 @@ const Behavior = MarionetteObject.extend({
   destroy() {
     this.stopListening();
 
+    if (this.view) {
```

Comment:
Yeah.. tests fail without it, but I think you're right and tests aren't using a view or something correctly..  looking into it now.